### DISCUSSION TITLE
Terminal/CLI-based front end?

### DISCUSSION THREAD

**Discussion by beyarkay** [monkeytypegame/monkeytype#4697] [Ideas]
Would a terminal frontend be considered out of scope for monkeytype? It would be really nice to do a typing test (with all of monkeytype's features) from my terminal.

**Comment by Miodec** [comment]
I've got no clue how you access a frontend from a terminal?

**Reply by fehmer** [reply]
`telnet monkeytype.com` or `ssh monkeytype.com`

Would be a cool feature but also a lot of work...

**Reply by unrealapex** [reply]
Not the answer you're looking for but I would recommend taking a look at [Wish](https://github.com/charmbracelet/wish).

demo:
```
ssh git.charm.sh
```
